lib/src/configuration.dart

```diff
@@ -92,9 +92,10 @@ abstract class Configuration implements Finalizable {
     this.fifoFilesFallbackPath,
     this.encryptionKey,
   }) {
-    if (encryptionKey != null && encryptionKey!.length != realmCore.encryptionKeySize) {
-      throw RealmException("EncryptionKey must be 64 bytes");
+    if (encryptionKey != null && encryptionKey!.isNotEmpty && encryptionKey?.length != realmCore.encryptionKeySize) {
+      throw RealmException("Wrong encryption key size (must be 0 or ${realmCore.encryptionKeySize})");
```

Comment:
Why do we accept empty keys here?

Reply:
They do the same in C-API.
I just do the same validation in the Configuration constructor, since we call the core function later on creating a realm with this config.
```
 if (key_size != 0 && key_size != 64) {
     throw std::logic_error{"Wrong encryption key size (must be 0 or 64)"};
 }
```

Reply:
OK. I agree.